### PR TITLE
084 hotfix + bool input validation

### PR DESCRIPTION
**Pull request by dITySoftware**
# Make sure these boxes are signed before submitting your Pull Request -- thank you.

# Must haves
- [x] I have read and followed the contributing guide lines at https://github.com/ait-aecid/logdata-anomaly-miner/wiki/Git-development-workflow
- [ ] Issues exist for this PR
- [ ] I added related issues using the "Fixes #<issue-id>"-notations
- [x] This Pull-Requests merges into the "development"-branch

Fixes #

# Submission specific

- [ ] This PR introduces breaking changes
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed

# Describe changes:
- mypy errors
- added bool input validation

### CODE REVIEW

File: aecid-testsuite/unit/parsing/AnyByteDataModelElementTest.py [hunk 1]
```diff
@@ -94,6 +98,7 @@ def test6get_match_element_match_context_input_validation(self):
         self.assertRaises(AttributeError, model_element.get_match_element, self.path, data)
         self.assertRaises(AttributeError, model_element.get_match_element, self.path, data.decode())
         self.assertRaises(AttributeError, model_element.get_match_element, self.path, 123)
+        self.assertRaises(AttributeError, model_element.get_match_element, self.path, 123)
```
Comment (landauermax) [inline aecid-testsuite/unit/parsing/AnyByteDataModelElementTest.py:101]: Why is this line here? It is the same as above?

Comment (dITySoftware) [reply aecid-testsuite/unit/parsing/AnyByteDataModelElementTest.py]: fixed that.

**Review by landauermax — APPROVED** [review]
Looks good to me
